test/cypress/integration/card/moto-mask-payment.spec.js

```diff
@@ -0,0 +1,207 @@
+const cardPaymentStubs = require('../../utils/card-payment-stubs')
+const gatewayAccountId = 42
+
+describe('Standard card payment flow', () => {
+  const tokenId = 'be88a908-3b99-4254-9807-c855d53f6b2b'
+  const chargeId = 'ub8de8r5mh4pb49rgm1ismaqfv'
+
+  const validPayment = {
+    cardNumber: '4444333322221111',
+    expiryMonth: '01',
+    expiryYear: '30',
+    name: 'Valid Paying Name',
+    securityCode: '012',
+    addressLine1: '10 Valid Paying Address',
+    city: 'London',
+    postcode: 'E1 8QS',
+    email: 'validpayingemail@example.com'
+  }
+
+  const createPaymentChargeStubsNoMoto = cardPaymentStubs.buildCreatePaymentChargeStubs(
+    tokenId, chargeId, 'en', gatewayAccountId, {}, {}, {
+      allowMoto: false
+    })
+
+  const createPaymentChargeStubsMotoMaskCardNumberOnly = cardPaymentStubs.buildCreatePaymentChargeStubs(
+    tokenId, chargeId, 'en', gatewayAccountId, {}, {}, {
+      allowMoto: true,
+      motoMaskCardNumberInput: true
+    })
+
+  const createPaymentChargeStubsMotoMaskSecurityCodeOnly = cardPaymentStubs.buildCreatePaymentChargeStubs(
+    tokenId, chargeId, 'en', gatewayAccountId, {}, {}, {
+      allowMoto: true,
+      motoMaskCardSecurityCodeInput: true
+    })
+
+  const createPaymentChargeStubsMotoMaskCardNumberAndSecurityCode = cardPaymentStubs.buildCreatePaymentChargeStubs(
+    tokenId, chargeId, 'en', gatewayAccountId, {}, {}, {
+      allowMoto: true,
+      motoMaskCardNumberInput: true,
+      motoMaskCardSecurityCodeInput: true
+    })
```

Comment:
Just thought — can we have a test `allowMoto: false`, `motoMaskCardNumberInput: true` and `motoMaskCardSecurityCodeInput: true` to check it doesn’t do any masking?

Reply:
Good idea - just added it